crates/aptos/src/node/local_testnet/mod.rs

```diff
@@ -248,6 +268,13 @@ impl CliCommand<()> for RunLocalTestnet {
             health_checkers.clone(),
         )?));
 
+        // Collect steps to run on shutdown. We run these in reverse.
+        let shutdown_steps: Vec<Box<dyn ShutdownStep>> = managers
+            .iter()
+            .flat_map(|m| m.get_shutdown_steps())
+            .rev()
+            .collect();
```

Comment:
nit - Is it possible to put this block into `get_shutdown_steps()`, so we can just call and get the steps in the correct order? Would be much cleaner I think

Reply:
To clarify, this reverses the order of all the shutdown steps. Doing what you suggest would only effect the order of shutdown steps within the return value of each ServiceManager.

Also the reversing behaviour isn't that important, it's just a little more logical.

Reply:
Ahh thats right, please ignore me lol